src/posit/connect/content.py

```diff
@@ -27,6 +29,13 @@ def __getitem__(self, key: Any) -> Any:
             return ContentItemOwner(params=self.params, **v)
         return v
 
+    @property
+    def oauth_associations(self) -> ContentItemAssociations:
```

Comment:
Maybe this should be expressed as `content.oauth.associations`?

Reply:
I like that alot better, yeah. Would the oauth property just be a passthrough class that we hang the associations resource off of?